### PR TITLE
Feature/iteration1

### PR DESCRIPTION
**Pull request by dstavis**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -2,9 +2,12 @@ class Color {
   constructor() {
     this.hexcode = this.#randomHexCode();
     this.locked = false;
-    this.id = Date.now().toString(); //id no work, fix me
+    this.id = Color.colorsMadeSoFar + 1;
```
Comment (dstavis) [inline main.js:5]: Do we want to use a different naming convention here? colorsMadeSoFar is a bit.......on the nose. What do you think? @aspitz1 @tfettig22

Comment (aspitz1) [reply main.js:5]: I don't mind it being on the nose. It's easy to understand what it does.

**Comment by aspitz1** [comment]
@tfettig22 I have reviewed this and I think It is good to give a merge. Unless we want to wait to merge it after iteration 1 is complete. Looks like we need to be able to do a couple more things. Though I am guessing we are done building the classes.

**Comment by aspitz1** [comment]
Actually... I it looks like we are still working within the Palette class. Should we wait to merge this branch til we complete it?

File: main.js (hunk 1 above)
Comment (tfettig22) [reply main.js:5]: Yeah I'm also good with the name, it does make it easy to understand what's happening

File: main.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+class Color {
+  constructor() {
+    this.hexcode = this.#randomHexCode();
+    this.locked = false;
+    this.id = Color.colorsMadeSoFar + 1;
```
Comment (dstavis) [inline main.js:5]: if we wanted C1, C2 type IDs we could write `this.id = "C" + Color.colorsMadeSoFar + 1;`

**Comment by dstavis** [comment]
> Actually... I it looks like we are still working within the Palette class. Should we wait to merge this branch til we complete it?

Yeah let's archive this PR and make a new one when we finish with Palette

**Pull request closed without merging**